src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1265,7 +1280,7 @@ private void DecodeCoClassAttribute(ref DecodeWellKnownAttributeArguments<Attrib
             }
         }
 
-        internal override bool IsComImport
+        internal sealed override bool IsComImport
```

Comment:
>IsComImport [](http://example.com/codeflow?start=38&length=11)

Can extension type be a ComImport type?

Reply:
Extension types can't have attributes at the moment. There's PROTOTYPE comment tracking that in the test file.
If you'd like I can leave PROTOTYPE comments to specific locations you're concerned about, to save review effort in later PR. Let me know

Reply:
> Extension types can't have attributes at the moment. There's PROTOTYPE comment tracking that in the test file.

If there is a special code that disallows attributes on extension types and disable their regular processing, then, I think it would be good to add a PROTOTYPE comment there listing the identified scenarios. Otherwise, I'd recommend marking each identified place separately.

Reply:
Attributes are currently disallowed on extension types because there's no attribute target that would correspond to extensions. 
Sure, I'll mark each place for when we get to support attributes.